content/en/logs/log_configuration/indexes.md

```diff
@@ -155,9 +155,9 @@ You can configure or remove this quota at any time when editing the Index:
 
 {{< img src="logs/indexes/daily_quota_config.png" alt="index details" style="width:70%;">}}
 
-An event is generated when either the daily quota or the warning threshold is reached:
+An event is generated when either the daily quota or the warning threshold is reached. To view these events, go to the <a href="https://app.datadoghq.com/event/explorer">Events Explorer</a> and search for `tags:log-management  "Warning threshold" OR "Daily quota"`. See example below:
```

Comment:
```suggestion
An event is generated when either the daily quota or the warning threshold is reached. To view these events, go to the [Events Explorer][22] and search for `tags:log-management  "Warning threshold" OR "Daily quota"`. See the example below:
```
At the bottom of the file, add this line:
`[22]: https://app.datadoghq.com/event/explorer`